LR_Tourist/TouristWebApp/Views/TravelVouchers/Create.cshtml

```diff
@@ -0,0 +1,63 @@
+﻿@model BLL.Model.TravelVoucher
```

Comment:
This view looks incorrect and without tag-helpers.

Reply:
fixed